app/models/suggestion.rb

```diff
@@ -0,0 +1,26 @@
+class Suggestion < ApplicationRecord
+  belongs_to :author , class_name: :User
+  has_many :votes , class_name: :ThumbsUp, foreign_key: :suggestion_id
+  has_many :voters , through: :votes, class_name: :User
```

Comment:
`class_name` cannot be utilized with a `through`. Need to use `source` with `through`